README.md

```diff
@@ -2,13 +2,9 @@
 
```

Comment:
All the *_D.png files do not appear to be used. They should be removed if not being used.

Reply:
Removed. Thanks for catching that!